docusaurus/src/pages/index.js

```diff
@@ -37,40 +43,113 @@ export default function Home() {
-            </div>
-            <OpenZitiHorizontalSection style={{fontSize:"large"}}>
-                <H1>The Mission</H1>
-                <p>OpenZiti is dedicated to making secure connectivity simple and accessible by <Highlight>replacing
-                    infrastructure with software</Highlight>. The world is software, your secure network needs to be software. OpenZiti is bringing
-                    zero trust networking principles directly inside applications through SDKs. The project provides
-                    everything you'll need to add zero trust principles to existing applications as well.
-                </p>
             </OpenZitiHorizontalSection>
-            <OpenZitiHorizontalSection style={{fontSize:"large"}}>
+            <OpenZitiHorizontalSection style={{fontSize:"large", padding: "0px"}}>
+
+                <div style={{display: "flex", flexDirection: "row", flexWrap: "wrap", justifyContent:"center"}}>
+
+                    <div className={styles.installChoice} style={{backgroundColor: "var(--openziti-callout-color)"}}>
+                        <H3>OpenZiti</H3>
+                        <p><b></b></p>
+                        <ul>
+                            <li>Open Source - Apache v2</li>
+                            <li>Host everything yourself</li>
+                            <li>Learn by doing</li>
+                            <li>Fully configurable</li>
+                            <li>Free forever</li>
+                        </ul>
+                        <div className={styles.installChoiceBtn} >
+                            <div style={{display: "flex", alignItems: "flex-start", justifyContent: "space-around"}}>
+                                <div className={styles.indexCtas}>
+                                    <Link className="button button--primary" to="#build-a-network">
+                                        <p>Get OpenZiti</p>
+                                    </Link>
+                                </div>
+                                <ThemedImage
+                                    alt="Docusaurus themed image"
+                                    sources={{
+                                        light: useBaseUrl('/img/github.svg'),
+                                        dark: useBaseUrl('/img/github-white.svg'),
+                                    }}
+                                    style={{display:"flex", alignItems:"center", height: "75px"}}
+                                />
+                            </div>
+                        </div>
+                    </div>
+
+                    <div className={styles.installChoice} style={{backgroundColor: "var(--zeds-callout-color)"}}>
+                        <H3>Ziti Edge Developer Sandbox</H3>
+                        <p><b></b></p>
+                        <ul>
+                            <li>Sandbox environment</li>
+                            <li>Easiest path for SDK-only solutions</li>
```

Comment:
should say "for SDK-only development", not solution (e2e solution will need to be on OZ or CZ - the sandbox is a sandbox, for learning/development of the app-embedded parts of the solution, not for real/production deployment).  

will also need to link to write-up/blog that goes into more detail on what it is / how to use it